Gruntfile.coffee

```diff
@@ -1063,7 +1082,7 @@ module.exports = (grunt) ->
 					flatten: true
 				,
 					cwd: "lib/MathJax"
-					src: [
+							src: [
```

Comment:
Oops, need to fix whitespace in this file again since I think the MathJax added that accidentally